app/models/Message.js

```diff
@@ -1,12 +1,16 @@
 var mongodb = require ('@onehilltech/blueprint-mongodb')
     ;
 
+var Organization = require ('./Organization')
+    ;
+
 var schema = new mongodb.Schema ({
-    sender_email:      {type: String, required: true, trim: true},
-    receiver_email:    {type: String, required: true, trim: true},
+    sender_email:   {type: String, required: true, trim: true},
+    receiver_email: {type: String, required: true, trim: true},
+    org_id:         {type: String, required: false, trim: true},
     received:       {type: Boolean, required: true},
     viewed:         {type: Boolean, required: true},
-    expireAt:        {type: Date, required: true},
+    expireAt:       {type: Date, required: false, default: Date.now},
```

Comment:
I am actually worried about this, I was trying to get the post from the user portal to work and it didn't like expireAt.  Thank you for reminding me.